backend/src/v5/processors/teamspaces/projects/federations/federations.js

```diff
@@ -0,0 +1,43 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { hasProjectAdminPermissions, isTeamspaceAdmin } = require('../../../../utils/permissions/permissions');
+const { getFavourites } = require('../../../../models/users');
+const { getFederations } = require('../../../../models/modelSettings');
+const { getProjectById } = require('../../../../models/projects');
+
+const Federations = {};
+
+Federations.getFederationList = async (teamspace, project, user) => {
```

Comment:
this is going to clash with @ChristosTsiotsias favourites PR as he did the same thing there.

But in general can we avoid duplicated code between fed and containers, any common code should go into a `common.js`?

... or we have a models folder instead, where we have federations.js and containers.js and a models.js or something.